src/Infrastructure/Resources/config/services.yaml

```diff
@@ -554,20 +554,27 @@ services:
     telemetry.command_event_validator:
         class: SprykerSdk\Sdk\Infrastructure\Event\Telemetry\TelemetryConsoleEventValidator
         arguments:
-            - !tagged_iterator telemetry.observable_command
+            - "@telemetry.observable_commands_class_name_collection"
 
     telemetry.command_event_listener:
         class: SprykerSdk\Sdk\Infrastructure\Event\Telemetry\TelemetryConsoleEventListener
         arguments:
             - "@telemetry.command_event_synchronizer"
             - "@telemetry.event_metadata_factory"
             - "@telemetry.command_event_validator"
-            - "@error_logger"
+            - "@error_logger_factory"
             - "%telemetry_enabled%"
         tags:
             - { name: kernel.event_listener, event: console.terminate }
             - { name: kernel.event_listener, event: console.error }
 
+    telemetry.observable_commands_class_name_collection:
+        class: SprykerSdk\Sdk\Infrastructure\Collection\TaggedClassNameCollection\TaggedClassNameCollection
+        arguments:
```

Comment:
Please, remove constructor from the `TaggedClassNameCollection` class and `arguments` definition from here. Replace it by specifying default value for the property.